### ISSUE TITLE
Expression preview field evaluates false value to empty

### ISSUE THREAD

**Issue by laurent-lemke** [n8n-io/n8n#3270]
**Describe the bug**
When we want to show a node from a past workflow  execution, field expression show an empty value if the referenced variable used has a false value :
 
<img width="1536" alt="Capture d’écran 2022-05-09 à 12 26 15" src="https://user-images.githubusercontent.com/32649054/167392383-b36881ca-aff5-4959-b7c7-27e24e5265e6.png">

However when we open the field expression, the expression is correctly evaluated and the false value is indeed shown. So the impact is minor but might confused at the first time.

**To Reproduce**
1. Run the below workflow
2. Click on the HTTP request node
3. The field expression related to the body parameter has a wrong value (one of its attribute is displayed as empty)
```
{
  "name": "My workflow 3",
  "nodes": [
    {
      "parameters": {},
      "name": "Start",
      "type": "n8n-nodes-base.start",
      "typeVersion": 1,
      "position": [
        -40,
        -80
      ]
    },
    {
      "parameters": {
        "functionCode": "item.falseVar = false;\nitem.trueVar = true;\n\nreturn item;"
      },
      "name": "FunctionItem",
      "type": "n8n-nodes-base.functionItem",
      "typeVersion": 1,
      "position": [
        240,
        -80
      ]
    },
    {
      "parameters": {
        "requestMethod": "POST",
        "url": "http//www.google.fr",
        "options": {},
        "bodyParametersUi": {
          "parameter": [
            {
              "name": "abc",
              "value": "={ \"x\": {{$json[\"falseVar\"]}}, \"y\": {{$json[\"trueVar\"]}}}"
            }
          ]
        }
      },
      "name": "HTTP Request",
      "type": "n8n-nodes-base.httpRequest",
      "typeVersion": 1,
      "position": [
        460,
        -80
      ]
    }
  ],
  "connections": {
    "Start": {
      "main": [
        [
          {
            "node": "FunctionItem",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "FunctionItem": {
      "main": [
        [
          {
            "node": "HTTP Request",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "HTTP Request": {
      "main": [
        []
      ]
    }
  },
  "active": false,
  "settings": {},
  "id": 8
}
```

**Expected behavior**
Display the right value

**Environment (please complete the following information):**
 - OS: [e.g. Ubuntu Linux 18.04]
 - n8n Version [e.g. 0.119.0]
 - Node.js Version [e.g. 14.16.0]
 - Database system [e.g. SQLite; n8n uses SQLite as default otherwise changed]
 - Operation mode [e.g. own; operation modes are `own`, `main` and `queue`. Default is `own`]

**Additional context**

**Comment by AilurusFulgenz** [comment]
Any news about it ? It is a quite annoying bug since we are not able to send false value in JSON.

**Comment by menasbeshay** [comment]
@laurent-lemke I'm facing same issue and I did a workaround for it by adding a new string field that holds the boolean value in string format (you can make that using Set node) :
`item.json.stringField = item.json.boolField ? "true" : "false";`
then use this string field in your expressions

**Comment by wieseljonas** [comment]
Experiencing the same issue. False always evaluates to empty when using in graphQL node variables

**Comment by Joffcom** [comment]
Just adding that the internal tracking reference for this one is `N8N-5967`.

**Comment by Joffcom** [comment]
Hey There,

This issue has now been fixed and from n8n `0.219.1` everything should be working as expected. I am going to mark this one as closed for now if you see the issue again let me know.
